controller/internal/enforcer/nfqdatapath/diagnostics_tcp_windows.go

```diff
@@ -0,0 +1,124 @@
+// +build windows
+
+package nfqdatapath
+
+import (
+	"fmt"
+	"math/rand"
+	"net"
+	"syscall"
+	"unsafe"
+
+	"github.com/aporeto-inc/gopkt/layers"
+	"github.com/aporeto-inc/gopkt/packet/ipv4"
+	"github.com/aporeto-inc/gopkt/packet/raw"
+	"github.com/aporeto-inc/gopkt/packet/tcp"
+	enforcerconstants "go.aporeto.io/trireme-lib/controller/internal/enforcer/constants"
+	"go.aporeto.io/trireme-lib/controller/internal/windows/frontman"
+	//"github.com/aporeto-inc/gopkt/routing"
+)
+
+type diagnosticsConnection struct {
+	SourceIp   net.IP
+	DestIp     net.IP
+	SourcePort uint16
+	DestPort   uint16
+	IgnoreFlow uint8
+}
+
+func createConnection(srcIP, dstIP net.IP) (*diagnosticsConnection, error) {
+	return &diagnosticsConnection{}, nil
+}
+
+func (diagConn *diagnosticsConnection) Close() {
+}
+
+func IPtoUint32Array(ip net.IP) [4]uint32 {
```

Comment:
I would make this private, and put some kind of comment about what it's doing (putting the ip address in a format that the driver expects).

Reply:
Will do.